data_structures/Stacks/Stock-Span-Problem.py

```diff
@@ -37,7 +38,7 @@ def calculateSpan(price, S):
 # A utility function to print elements of array 
 def printArray(arr, n): 
     for i in range(0,n): 
-        print arr[i], 
+        print (arr[i],end =" ") 
```

Comment:
Please delete the space...  __print (arr[i],end =" ")__ --> __print(arr[i], end=" ")__

__print()__ is now a function like all other functions so it should be formatted in the same way.